draft-opsarea-rfc5706bis.md

```diff
@@ -870,6 +870,25 @@ Information Models and Data Models
      the L3VPN Service Model (L3SM) {{?RFC8299}} and the L2VPN Service Model (L2SM)
      {{?RFC8499}}.
 
+  A common challenge in YANG model development lies in defining the
+  relationships between abstract service or network constructs and the
+  underlying device models. Therefore, when designing YANG modules, it
+  is important to go beyond simply modeling configuration and
+  operational data (i.e., leaf nodes), and also consider how the
+  status and relationships of abstract or distributed constructs can
+  be reflected based on parameters available in the network.
+
+  For example, the status of a service may depend on the operational state
+  of multiple network elements to which the service is attached. In such
+  cases, the YANG model (and its accompanying documentation) should
+  clearly describe how service-level status is derived from underlying
+  device-level information. Similarly, it is beneficial to define
+  notifications or events that signal changes in these underlying states,
```

Comment:
```suggestion
  events (and notifications) that trigger signaling changes in an underlying state,
```

Reply:
I think your suggested changes don't convey the same meaning as the original text.  Maybe:

> events (and notifications) that indicate changes in an underlying state

Reply:
@xorrkaz, yes as I wanted to insist on the triggered signals. Reworded to cover both aspects. Please check.

Reply:
Your new change reads better to me.